internal/config/config_test.go

```diff
@@ -15,199 +14,135 @@ import (
 )
 
 type mockReadFile struct {
-	contents   []byte
-	successful bool
+	contents []byte
 }
 
 func (m mockReadFile) readFile(name string) ([]byte, error) {
-	if !m.successful {
-		return nil, errors.New("an error")
-	}
-
 	if m.contents != nil {
-		return yaml.Marshal(m.contents)
+		return m.contents, nil
 	}
 
 	return yaml.Marshal(map[string]interface{}{
 		"access-token": "test-access-token",
 	})
 }
 
-func TestNewConfigFromFile(t *testing.T) {
```

Comment:
Removed these tests so the `TestService_VerifyAccessToken` tests seems like they were updated I guess.